apps/desktop/desktop/src/main/java/bisq/desktop/main/content/bisq_easy/offerbook/MarketChannelItem.java

```diff
@@ -119,12 +119,13 @@ private void setUpColorAdjustments() {
         SELECTED_COLOR_ADJUST.setBrightness(-0.1);
     }
 
-    private void updateNumOffers() {
+    private void updateNumbers() {
         UIThread.run(() -> {
             int numOffers = (int) channel.getChatMessages().stream()
                     .filter(BisqEasyOfferbookMessage::hasBisqEasyOffer)
                     .count();
             getNumOffers().set(numOffers);
+            activateNotifications();
```

Comment:
I think a `applyNotification(notification)` should be enough. To unbind and rebind the observer seems to be wasteful and signals a different intention.

Reply:
I assumed you may have faced troubles with this and that was the quick fix you had found out as the same method is called for similar updates across BisqEasyOfferbookController (```updateFilteredMarketChannelItems``` and ```updateFavouriteMarketChannelItems```). So I took the 'safe approach' of following the same pattern. I'll change and test those code paths as well so that we remain consistent.

Reply:
Oh yes there its also not the best approach. Better to make the `applyNotification` method public and call that.